src/main/java/Joey.java

```diff
@@ -0,0 +1,137 @@
+import java.util.Scanner;
+
+public class Joey {
+    public static void main(String[] args) throws JoeyException {
```

Comment:
This is a very long method. Try to avoid long methods. Take corrective action if method exceeds 30 lines of code.